google-cloud-storage/src/test/java/com/google/cloud/storage/conformance/retry/RpcMethodMappings.java

```diff
@@ -1795,7 +1796,8 @@ private static void insert(ArrayList<RpcMethodMapping> a) {
                 .build());
         a.add(
             RpcMethodMapping.newBuilder(119, objects.insert)
-                .withApplicable(TestRetryConformance::isPreconditionsProvided)
+                .withApplicable(
+                    and(TestRetryConformance::isPreconditionsProvided, not(groupIsResumableUpload)))
```

Comment:
so this is here because we cannot have both preconditions and resumable uploads? sorry if this is naive I do not really get what this is doing.